release/license_info.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# Copyright (C) 2018 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# shellcheck source=common.sh
+
+set -e
+
+SCRIPT_DIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
+. "${SCRIPT_DIR}/../common.sh"
+. ./config/config.sh
+
+var_load MIX_VERSION
+var_load CLR_LATEST
+dnf_conf="${MIXER_DIR}/.yum-mix.conf"
+
+generate_license_info() {
+    awk '{print $1}' < "${MIXER_DIR}/update/image/${MIX_VERSION}/os-packages" |
+        xargs dnf --quiet --config "${dnf_conf}" --releasever="${CLR_LATEST}" --installroot=dnf-cache repoquery --queryformat "%{LICENSE}" |
+        tr ' ' '\n' |
+        sort -u
+}
+
+pushd "${WORK_DIR}" > /dev/null
+echo "Generating License Info"
+generate_license_info > "${PKG_LICENSES_FILE}"
```

Comment:
This is even better!

I would just replace `s/"${PKG_LICENSES_FILE}"/"${WORK_DIR}/${PKG_LICENSES_FILE}"/` and completely remove the need to push/pop into the WORK_DIR folder.

There is only two references to "${PKG_LICENSES_FILE}", so it should be easy. Either that or have a variable: `output_file="${WORK_DIR}/${PKG_LICENSES_FILE}"`